src/app/space/app-launcher/import-app/import-app.component.ts

```diff
@@ -66,4 +72,20 @@ export class ImportAppComponent implements OnDestroy, OnInit {
   complete(): void {
     this.router.navigate(['/', this.loggedInUser.attributes.username, this.currentSpace.attributes.name]);
   }
+
+  addQuery(): QueryJson {
+    const query = '{\"application\":[\"' + this.projectile.sharedState.state.projectName + '\"]}';
```

Comment:
`queryparams` can accept an object. This can be simplified to:
```
    return {
      application: this.projectile.sharedState.state.projectName
    };
```

Reply:
In the pipeline page, a user can filter more than one application at a time. So, it accepts an array of the application name.

Reply:
My mistake. But the approach to return an object still works, no? Or is that why you have to return a string?